linkedin-clone/src/components/Main.js

```diff
@@ -30,13 +30,13 @@ const Main = (props) => {
 
   const history = useHistory();
 
-  const handleApply = (jobPostingId) => {
-    history.push(`/job-posting/${jobPostingId}`);
-  };
+  // const handleApply = (jobPostingId) => {
+  //   history.push(`/job-posting/${jobPostingId}`);
```

Comment:
applying still not implemented ?

Reply:
applying is implemented in JobPostingPage.js
https://raw.githubusercontent.com/TNT05/MiniCap/main/linkedin-clone/src/components/JobPostingPage.js
The button only takes to the job posting page.